src/main/java/com/julienvey/trello/impl/TrelloUrl.java

```diff
@@ -48,6 +48,11 @@
     public static final String GET_ORGANIZATION_BOARD = "/organizations/{organizationId}/boards?";
     public static final String GET_ORGANIZATION_MEMBER = "/organizations/{organizationId}/members?";
 
+    public static final String CREATE_LABEL = "/labels?";
```

Comment:
3 constants pointing to the same string. I suggest following naming convention, despite the fact that it will stand out from the `TrelloUrl` naming conventions.

- `/labels?` is a `LABEL_RESOURCE` or just `LABEL`
- `/labels/labelId?` is a `SPECIFIC_LABEL_RESOURCE` or just `SPECIFIC_LABEL`

We make use of REST API but we name our constants like we would use SOAP.

Reply:
let's look at renaming labels in a separate PR to limit the amount of changes